lib/tasks/expire.js

```diff
@@ -1,8 +1,21 @@
 const moment = require('moment');
 
-module.exports = ({ Project }) => () => {
-  return Project.query()
-    .update({ status: 'expired' })
-    .where('expiryDate', '<=', moment().toISOString())
-    .where('status', 'active');
+module.exports = ({ Project }) => async () => {
+  const midnightLastNight = moment.utc().startOf('day').toISOString();
```

Comment:
using UTC because otherwise `startOf('day')` will actually be `23:00` the night before?!